Tests/TestTools.cpp

```diff
@@ -11,14 +11,16 @@
 
 #include "Tests/TestTools.h"
 
-boost::property_tree::ptree
-readXml(const char xml[])
+namespace Tests
+{
+boost::property_tree::ptree readXml(const char xml[])
 {
     boost::property_tree::ptree ptree;
     std::istringstream xml_str(xml);
     boost::property_tree::read_xml(
-        xml_str, ptree,
-        boost::property_tree::xml_parser::no_comments |
-        boost::property_tree::xml_parser::trim_whitespace);
+        xml_str, ptree, boost::property_tree::xml_parser::no_comments |
+                            boost::property_tree::xml_parser::trim_whitespace);
```

Comment:
this formatting is still rather random. Which clang version do you use?

Reply:
clang-format version 10.0.0

Reply:
Ran several times, but that ugly format does not change.

Reply:
Should be fixed. Actually there is no change in those lines, remove them from the PR.

Reply:
Recovered to the previous format.